src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/WorkflowRunControllerTests.kt

```diff
@@ -97,10 +97,124 @@ class WorkflowRunControllerTests
             on { getWorkflowRunDetails("fakeKey") } doThrow UnknownObjectError("key", "workflow")
         }
 
-        val sut = WorkflowRunController(mockContext, mockRepo)
+        val sut = WorkflowRunController(mockContext, mockRepo, mock())
 
         Assertions.assertThatThrownBy { sut.getWorkflowRunDetails() }
-                .isInstanceOf(UnknownObjectError::class.java)
-                .hasMessageContaining("Unknown workflow : 'key'")
+            .isInstanceOf(UnknownObjectError::class.java)
+            .hasMessageContaining("Unknown workflow : 'key'")
     }
+
+    @Test
+    fun `can deserialize a workflow description`()
```

Comment:
This test mainly serves to placate Codecov w.r.t. data classes but also provides a full JSON workflow example